src/components/Tooltip/Tooltip.tsx

```diff
@@ -17,6 +17,8 @@ export interface TooltipProps {
   light?: boolean;
   /** Toggle whether the tooltip is visible */
   active?: boolean;
+  /** Dismiss Tooltip once pointer is no longer over children */
+  preventInteraction?: boolean;
```

Comment:
Personally, I would write this as:
`preventInteraction?: TooltipOverlayProps['preventInteraction'];`